net/redisclient_test.go

```diff
@@ -100,13 +100,13 @@ type addressUpdater struct {
 // 5: [foo bar]
 // 6: [foo bar baz]
 // ...
-func (u *addressUpdater) update() []string {
+func (u *addressUpdater) update() ([]string, error) {
```

Comment:
Where do we need this change?

Reply:
`redisclient_test.go  L:148`  as AddrUpdater is updated to `func () ([] string, error) `

Reply:
ok